### PR TITLE
[nikobus] - added new trigger channels for Nikobus push button

### PR DESCRIPTION
**Pull request by crnjan**
When a physical Nikobus push button is presssed&hold, it will send a new packet every ~50ms. Nikobus v2 binding simply forwards this events while Nikobus v1 filtered and converted them to short/long press(es). In order to add missing functionality (also requested by the community) new trigger channels are introduced to bridge this gap. 

Signed-off-by: Boris Krivonog <boris.krivonog@inova.si>

### CODE REVIEW

File: bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/config/config.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		<parameter name="command" type="text">
+			<label>Command</label>
+			<description>Command to send</description>
+		</parameter>
+		<parameter name="delay" type="integer" required="true" min="0">
+			<label>Delay</label>
+			<description>Delay in milliseconds before triggered</description>
+			<default>0</default>
+		</parameter>
+		<parameter name="period" type="integer" min="0">
```
Comment (cpmeister) [inline bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/config/config.xml:17]:
```suggestion
		<parameter name="period" type="integer" min="0" unit="ms">
```

File: bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/config/config.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="push-button:trigger-filter:config">
+		<parameter name="command" type="text">
+			<label>Command</label>
+			<description>Command to send</description>
+		</parameter>
+		<parameter name="delay" type="integer" required="true" min="0">
```
Comment (cpmeister) [inline bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/config/config.xml:12]:
```suggestion
		<parameter name="delay" type="integer" required="true" min="0" unit="ms">
```

File: bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/config/config.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+			<default>0</default>
+		</parameter>
+		<parameter name="period" type="integer" min="0">
+			<label>Period</label>
+			<description>Time in milliseconds between successive triggers</description>
+		</parameter>
+	</config-description>
+
+	<config-description uri="push-button:trigger-button:config">
+		<parameter name="threshold" type="integer" required="true" min="0">
```
Comment (cpmeister) [inline bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/config/config.xml:24]:
```suggestion
		<parameter name="threshold" type="integer" required="true" min="0" unit="ms">
```

File: bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/thing/pc-link.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+		<description>PC-Link via serial connection</description>
+
+		<config-description>
+			<parameter name="port" type="text" required="true">
+				<label>Port</label>
+				<context>serial-port</context>
+				<limitToOptions>false</limitToOptions>
+				<description>The serial port used to connect to the Nikobus PC Link.</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" max="65535" min="10">
```
Comment (cpmeister) [inline bundles/org.openhab.binding.nikobus/src/main/resources/OH-INF/thing/pc-link.xml:18]:
```suggestion
			<parameter name="refreshInterval" type="integer" max="65535" min="10" unit="s">
```

**Review by cpmeister — CHANGES_REQUESTED** [review]

**Comment by crnjan** [comment]
Accidentally pushed an unwanted commit, should be ready for review now.

**Review by cpmeister — CHANGES_REQUESTED** [review]
Please put nested classes at the end of the file. But other than that LGTM.

**Comment by crnjan** [comment]
Build failure should not be related to this PR

```
[INFO] openHAB Add-ons :: Bundles :: Nikobus Binding ...... SUCCESS [02:43 min]
```

**Pull request merged**